packages/sui-ssr/server/ssr/index.js

```diff
@@ -165,9 +168,16 @@ export default async (req, res, next) => {
 
   const {__HTTP__} = initialProps
   if (__HTTP__) {
-    const {redirectTo} = __HTTP__
+    const {redirectTo, redirectStatusCode} = __HTTP__
```

Comment:
I think that the `redirect` doesn't work in Client-Side. Any plan to add support in the future ?

Reply:
Yup! would be nice since `getInitialProps` is intended to be isomorphic